ephios/core/templates/core/group_list.html

```diff
@@ -14,25 +14,54 @@ <h1>{% trans "Groups" %}</h1>
         <a class="btn btn-secondary mb-3" href="{% url "core:group_add" %}"><span
                 class="fa fa-plus"></span> {% trans "Add group" %}</a>
     {% endif %}
-    <table id="event_table" class="table table-striped display">
-        <thead>
-        <tr>
-            <th>{% trans "Name" %}</th>
-            <th>{% trans "Action" %}</th>
-        </tr>
-        </thead>
-        <tbody>
+    <ul class="list-group mb-3">
         {% for group in group_list %}
-            <tr>
-                <td>{{ group.name }}</td>
-                <td class="d-flex">
-                    <a class="btn btn-secondary btn-sm text-nowrap" href="{% url "core:group_edit" group.pk %}"><span
-                            class="fa fa-edit"></span> <span class="d-none d-md-inline">{% trans "Edit" %}</span></a>
-                    <a class="btn btn-secondary btn-sm text-nowrap ms-1" href="{% url "core:group_delete" group.pk %}"><span
-                            class="fa fa-trash-alt"></span> <span class="d-none d-md-inline">{% trans "Delete" %}</span></a>
-                </td>
-            </tr>
+            <li class="list-group-item">
+                <div class="row mb-0">
+                    <div class="col-12 col-lg-4">
+                        <h5 class="d-inline">
+                            {{ group.name }}
+                        </h5>
+                        <div id="popover-title-{{ group.id }}" class="d-none"></div>
+                        <div id="popover-content-{{ group.id }}" class="d-none">
+                            {% for user in group.user_set.all %}{% if not forloop.first %}, {% endif %}
+                                {{ user.get_full_name }}{% endfor %}
```

Comment:
did you test that with a large number of people? I think there are users that have all 100 users in a basic group and I think that could look quite odd

Reply:
![image](https://github.com/ephios-dev/ephios/assets/5970076/f8bb191f-0279-4e4d-9312-c3ae9c1ecaef)

well... maybe we cut it off at some point?

Reply:
> ![image](https://user-images.githubusercontent.com/5970076/264771259-f8bb191f-0279-4e4d-9312-c3ae9c1ecaef.png)
> 
> well... maybe we cut it off at some point?

yep, definitely need to do that. Maybe at 10?